### PR TITLE
docs: add instructions for mongodb setup

### PR DESCRIPTION
**Pull request by RafaelVidaurre**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [X] I linked the issues related to this PR in its description (if any).
- [X] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,36 @@ This is a monorepo containing the webapps and libraries in Polymath. In each pac
...
+
+```
+yarn install
+```
+
+## Installing MongoDB
+
+You will need to run MongoDB in your machine to run the dApps locally
+
+Installing mongodb through with [Homebrew](https://brew.sh/)
```
Comment (josepmc) [inline README.md]: This only works in OS X, in mac you'll just do `apt install mongodb`

File: README.md [hunk 2]
```diff
@@ -14,7 +14,36 @@ This is a monorepo containing the webapps and libraries in Polymath. In each pac
...
+```
+
+Mongodb uses `/data/db` as its data folder. If you don't have it you must create
+it
+
+```
+mkdir -p /data/db
+```
+
+Make sure the current user has read/write access to the `/data/db` directory
```
Comment (josepmc) [inline README.md]: chown / chmod perhaps?

File: README.md (hunk 1 above)
Comment (RafaelVidaurre) [reply README.md]: @josepmc Yeah, I guess I should've mentioned this are OSX-specific install instructions

File: README.md (hunk 2 above)
Comment (RafaelVidaurre) [reply README.md]: 👍 !

File: README.md [hunk 3]
```diff
@@ -14,7 +14,42 @@ This is a monorepo containing the webapps and libraries in Polymath. In each pac
...
+```
+brew update
+brew install mongodb
+```
+
+Mongodb uses `/data/db` as its data folder. If you don't have it you must create
+it
+
+```
+mkdir -p /data/db
```
Comment (Grsmto) [inline README.md]: I had to `sudo` for this command to run.

Comment (RafaelVidaurre) [reply README.md]: Gotcha, adding this

**Pull request merged**